notebooks/week3/02_model_serving.py

```
@@ -0,0 +1,190 @@
+    ),
+)
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC # Call the Endpoint
+
+# COMMAND ----------
+
+# DBTITLE 1,set token and host
+token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff

Undefined name `dbutils`

(F821)